### ISSUE TITLE
Accessibility testing issues

### ISSUE THREAD

**Issue by at48452** [kamranahmedse/driver.js#434]
Hi there
I am using cypress-axe accessibility testing (WCAG AA) to check the new tutorial I've written using driver.js. The driver.js code is failing accessibility in a number of areas (screenshots attached):

1. header tag for the title in the popover leads to duplicate banner landmark, and I don't want to remove the header tag around the main page header, as this is a more important landmark for screen-reader users
2. same element does not have a unique role/label/title
3. div with id driver-popover content is missing an aria label
4. div with id-driver-dummy-element has an aria-expanded attribute, despite not being a collapsible menu

I've yet to solve issue 1 as manually changing the role of the element wasn't effective. Issues 2 and 3 were solved relatively easily by accessing element attributes using the onPopoverRender method on the driver object. I had to resort to deleting the whole div in issue 4. It didn't seem to affect the functionality of driver, so I'd be interested to know what purpose that part of the code serves? 
I'd appreciate a fix for issue 1 in particular.

Otherwise, I've found driver.js very easy to work with and the supporting docs excellent :)

Thanks

![Screenshot 2023-09-19 at 15 56 12](https://github.com/kamranahmedse/driver.js/assets/141840279/a1465114-7f05-4fc9-a205-3c3caefabf97)
![Screenshot 2023-09-19 at 15 56 38](https://github.com/kamranahmedse/driver.js/assets/141840279/ce6a4225-8fe2-480a-88fe-ac9114078f73)
![Screenshot 2023-09-19 at 15 57 05](https://github.com/kamranahmedse/driver.js/assets/141840279/917e72b2-1d52-4f34-9056-dd0a0505ddc4)

![Screenshot 2023-09-19 at 15 57 46](https://github.com/kamranahmedse/driver.js/assets/141840279/34959a12-2cb4-4e5b-a075-f6214d71299f)
